pytest_catchlog/fixture.py

```diff
@@ -16,6 +16,9 @@ class LogCaptureFixture(object):
     def __init__(self, item):
         """Creates a new funcarg."""
         self._item = item
+        # set logging levels to facilitate use without needing to import logging
+        for level in ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']:
+            setattr(self, level, getattr(logging, level))
```

Comment:
Generic of course is also a good thing...
See update, slightly different check but otherwise the same as your proposal.